docs/daikon/census_divisions.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: census-divisions/
+parent: Daikon REST API
+nav_order: 5
+---
+
+Census division is the general term for a Canadian geographic subregion (such as county, municipalité régionale de comté and regional district). They are typically established to facilitate the provision of public services (such as police or ambulance), but StatCan also utilizes these regions to help disseminate and organize statistical data.
+
+<style>
+td, th {
+   border: none!important;
+}
+</style>
+
+### GET /census-divisions/search ###
+{: .d-inline-block }
+
+Private Beta
+{: .label .label-purple }
+
+Returns basic attributes, such as name and type, of a given census division or all census divisions within a region. 
+
+| Request Parameter      | Details |
+| ----------- | ----------- |
+| region_code      | (Optional) The two letter internationally standardized ISO-3166-2 region code. Returns all census divisions in Canada if `null`|
+| census_division_name | (Optional) Census division name| 
+
+
+| Response Key      | type | Definition |
+| ----------- | ----------- |----------- |
+| id      | UUID | The unique 4-digit census division id (first two digits correspond to region) |
+| dguid      | string | [Dissemination Geographic Unique Identifier](https://www12.statcan.gc.ca/census-recensement/2021/ref/dict/az/definition-eng.cfm?ID=geo055) |
+| census_division_name      | string | Census division name |
+| census_division_type      | string | Census division type |
+| land_area     | numeric(32,2) | Land area of census division, in square kilometers (km<sup>2</sup>) |
+| region_code   | string | The two letter internationally standardized ISO-3166-2 region code |
+
+#### Example Requests ####
+<br>
+Specifying a region using the `region_code` will return all census divisions in that region.<br><br>
+`https://api.tm41m.com/census-divisions/search?region_code=ON`
+{: .d-inline-block }
+GET
+{: .label .label-blue }
+
+##### Response: #####
+
+```json
+{
+    "region_code": "ON"
+}
+```
+The attributes of a single census division can be observed by specifying the `census_division_name`. Special characters such as spaces and accents can be passed unencoded to the request.<br><br>
+`https://api.tm41m.com/census-divisions/search?census_division_name=La Matapédia`
+{: .d-inline-block }
+GET
+{: .label .label-blue }
+
+##### Response: #####
+
+```json
+{
+    "census_division_name": "La Matapédia"
```

Comment:
You're going to alter the response json to include the keys in the table above right?

Reply:
> You're going to alter the response json to include the keys in the table above right?

Yes, they'll have all the same response keys as the actual api responses